pkgx.rb

```diff
@@ -33,6 +33,10 @@ def install
     if File.file? "Cargo.toml"
       system "cargo", "build", "--release"
       bin.install "target/release/pkgx"
+
+      system "curl" "-o", "pkgm", "https://pkgxdev.github.io/pkgm/pkgm.ts"
```

Comment:
there's a way for this... you'd have `pkgx` write the `@latest` version of `pkgm` into its release distribution (`echo foo >.pkgm.version`) then you could fetch it out of tree using the git tag. maybe not worth it until we start having version incompatibilities, but that's how I'd do it.